faqs/galaxy/workflows_get_id.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Get the workflow id
+area: workflows
+box_type: tip
+layout: faq
+contributors: [simonbray,wm75,lecorguille]
+---
+
+1. Go to the {%icon galaxy-workflows-activity %}workflows panel in Galaxy and find the workflow you've just created.
+2. Click on it to get display the Workflow Preview
+3. The URL in your browser will look something like `https://usegalaxy.eu/workflow/editor?id=34d18f081b73cb15`. Copy the part after `?id=` - this is the workflow ID, in our case: `34d18f081b73cb15`
```

Comment:
Hmm, for me the URL doesnt change when I get the workflow preview, is there another way to easily get the ID?